libs/barista-components/button-group/BUILD.bazel

```
@@ -16,7 +41,58 @@ filegroup(
     ]),
 )
 
+sass_library(
+    name = "scss_lib",
+    srcs = glob(
+        ["src/**/_*.scss"],
+    ),
+)
+
+sass_binary(
+    name = "button-group-styles",
+    src = "src/button-group.scss",
+    deps = [":scss_lib"],
+)
+
+sass_binary(
+    name = "button-group-item-styles",
+    src = "src/button-group-item.scss",
+    deps = [":scss_lib"],
+)
```

Use a multi sass binary for the styles

```suggestion
multi_sass_binary(
    name = "styles",
    srcs = [
        "src/button-group.scss",
        "src/button-group-item.scss",
        ":theme",
    ],
)
```